tests/h/search/query_test.py

```diff
@@ -689,115 +689,109 @@ def search(self, search):
 
 @pytest.mark.usefixtures("pyramid_config")
 class TestHiddenFilter:
```

Comment:
I actually couldn't work out what these tests were trying to assert or how they were doing it. I was just trying to get these tests passing again, but I couldn't work it out.

This is a refactor that I think is much more obvious in what it's asserting and how it does it.